source/shared/net/mobile-wallet/share-address-mw.rst

```diff
@@ -23,9 +23,10 @@ Share your account address
 
 #. Go to the **Accounts** page.
 
-#. Tap the balance of the account you want to find the address to.
+#. On the account you want to find the address to tap **RECEIVE**.
 
-#. Tap **ADDRESS**.
+   .. image:: ../images/mobile-wallet/MW13.png
```

Comment:
This screenshot should probably also be without the backup warning at the top of the screen